### PR TITLE
release(v0.5.0-alpha.0): prepare release

### PR DESCRIPTION
**Pull request by Unix4ever**
This is the official v0.5.0-alpha.0 release.

Signed-off-by: Artem Chernyshev <artem.chernyshev@talos-systems.com>

### CODE REVIEW

File: hack/release.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ github_repo = "talos-systems/sidero"
 match_deps = "^github.com/(talos-systems/[a-zA-Z0-9-]+)$"
 
 # previous release
-previous = "v0.3.2"
+previous = "v0.4.1-beta.0"
```
Comment (smira) [inline hack/release.toml:9]: nit: I think it should be v0.4.0, the last tag which is shared on master and `release-0.4`

Comment (Unix4ever) [reply hack/release.toml:9]: rgr

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,408 @@
...
+* **go.uber.org/zap**                                                    v1.19.0 **_new_**
+* **golang.org/x/net**                                                   853a461950ff -> 6635138e15ea
+* **golang.org/x/sys**                                                   39ccf1dd6fa6 -> 51b60fd695b3
+* **golang.zx2c4.com/wireguard/wgctrl**                                  0073765f69ba **_new_**
+* **google.golang.org/grpc**                                             v1.41.0 -> v1.42.0
+* **inet.af/netaddr**                                                    c74959edd3b6 **_new_**
+
+Previous release can be found at [v0.4.0](https://github.com/talos-systems/sidero/releases/tag/v0.4.0)
+
+## [Sidero 0.4.0](https://github.com/talos-systems/sidero/releases/tag/v0.4.0) (2021-10-18)
```
Comment (smira) [inline CHANGELOG.md:196]: something wrong here? too many releases?

**Review by smira — APPROVED** [review]

**Pull request merged**